pkg/logger/s3_log_writer.go

```diff
@@ -36,13 +40,38 @@ func CreateBucket(minioClient *minio.Client, bucketName string) error {
 }
 
 func (l *S3LogWriter) putLog(msg string) {
+	now := time.Now()
+
+	level := "info"
+
+	if len(msg) > 0 {
+		if strings.HasPrefix(msg, "✗ ") {
+			level = "error"
+		} else if strings.HasPrefix(msg, "⚠️ ") {
+			level = "warning"
+		}
+	}
+
+	result := &pb.LogEntry{
+		Timestamp: now.Format(time.RFC3339),
```

Comment:
:+1:

Reply:
I wasn't sure which timestamp format you need for this endpoint, so copied it from the timestamp field in another endpoint. Glad it is OK.